documentation/course/e2/README.md

```diff
@@ -679,38 +697,49 @@ TODO @piterek130
     <th>Wady</th>
   </tr>
   <tr>
-    <th>Rozwiązanie 1</th>
+    <th>Single Page Application (SPA)</th>
     <td>
       <ul>
-        <li>Zaleta 1</li>
+        <li>Wysoka wydajność</li>
```

Comment:
Why? If anything, there's lower performance than with server-rendered pages, since we have to first send all of the JS to the client and then also run it.